nilearn/plotting/js_plotting_utils.py

```diff
@@ -87,6 +87,7 @@ def colorscale(cmap, values, threshold=None, symmetric_cmap=True,
         vmin = 0
     if vmax is None:
         vmax = abs_values.max()
+    vmax = float(vmax)
```

Comment:
A comment here, why we are typecasting this to float, so future maintainers will have an easier time understanding it.